jujugui/static/gui/src/app/components/inspector/config/config.js

```diff
@@ -344,7 +346,11 @@ YUI.add('inspector-config', function() {
           <span>Choose Series</span>
           <select
             className="inspector-config__select"
+            disabled={hasRelations}
```

Comment:
It would be nice if we had some disabled styling for selects...I believe this has come up elsewhere as well.